bin/convert-argv.js

```diff
@@ -35,11 +35,11 @@ module.exports = function(optimist, argv, convertOptions) {
 	}
 
 	var configFileLoaded = false;
-	var configPath, ext;
+	var configFile = {};
```

Comment:
I can make `configFiles` always be an array to remove a lot of the if statements later. If there's only one options object in the array, I'll only export the options object. Otherwise the webpack will always use the MultiCompiler.

```
if (options.length === 1) {
  return processConfiguredOptions(options[0]);
} else {
  return processConfiguredOptions(options);
}
```